ClientApp/src/Pages/AdminSightingsList/AdminSightingsList.tsx

```diff
@@ -25,19 +35,20 @@ export function TableRow(data: Sighting): JSX.Element {
       <td>{data.submittedByName}</td>
       <td>{data.submittedByEmail}</td>
       <td>
-        <button type="button" className="btn btn-warning">
-          Review
-        </button>
+        <Link to={`/admin/confirm-sighting/${data.id}`}>
+          <button type="button" className="btn btn-warning" disabled={confirmClicked || deleteClicked} aria-disabled={confirmClicked || deleteClicked}>
```

Comment:
I just tried it, it seems to have coped okay... are you expecting it to break or do something weird?

Reply:
Yes I imagined it might send off multiple requests, however if we have separate delete and restore endpoints instead of a single one this issue doesn't matter (as the endpoints are now idempotent)

Reply:
yep they're separate now :) I've tried spamming both buttons and it seems to work as it should